build.sbt

```diff
@@ -387,6 +387,41 @@ lazy val sbtProj = (project in file("sbt"))
   )
   .configure(addSbtCompilerBridge)
 
+lazy val vscodePlugin = (project in file("vscode-sbt-scala"))
```

Comment:
Feel free to take some inspiration from https://github.com/lampepfl/dotty/blob/657385ba36eb138f1f78413311b7c108d3dc8ac2/project/Build.scala#L891-L936